bqn-mode.el

```diff
@@ -413,9 +422,14 @@ bqn-comint-process-session and echoes the result."
     (error "Attempt to evaluate empty region to %s" bqn-comint--process-name))
   (when (and bqn-comint-flash-on-send (pulse-available-p))
     (pulse-momentary-highlight-region start end))
-  (let ((region (buffer-substring-no-properties start end))
-        (process (get-buffer-process (bqn-comint-buffer))))
-    (message "%s" (bqn--comint-call-process-silently process region))))
+  (let* ((region (buffer-substring-no-properties start end))
+         (process (get-buffer-process (bqn-comint-buffer)))
+         (response (bqn--comint-call-process-silently process region)))
+    (if bqn-comint-use-overlay
+        (eros--make-result-overlay response
+          :where end
+          :duration eros-eval-result-duration)
```

Comment:
Adjusting the format might be desirable so that multiline arrays display nicely; suggestions welcome

Reply:
I think I found a relatively acceptable solution; one-line results are displayed like this: 
![2024-09-16-095847_158x87_scrot](https://github.com/user-attachments/assets/7989488e-6d12-484b-8773-baf4c8de6afa)
and as soon as we go multiline, we change to 
![2024-09-16-095859_227x443_scrot](https://github.com/user-attachments/assets/395a9e31-1e2f-4864-a9bf-5d894f0016df)

Plus, when called with a universal argument, the result is inserted into the buffer:
![2024-09-16-095913_192x99_scrot](https://github.com/user-attachments/assets/885d856c-531b-402d-b335-c0a0041008da)
![2024-09-16-095924_236x436_scrot](https://github.com/user-attachments/assets/0fd55a5f-ea27-40c9-861c-50bb180972dc)